app/scripts/lib/metrics.js

```diff
@@ -121,6 +122,7 @@ define([
     this._env = options.environment || new Environment(this._window);
 
     this._lastAbLength = 0;
+    this._startTime = options.startTime || Date.now();
```

Comment:
More on this, if the browser supports NavigationTiming, the `baseTime` will be from `navigationStart` instead of when the app starts.